helpers.go

```diff
@@ -33,10 +39,20 @@ func fromLibdns(zone string, records []libdns.Record) ([]leasewebRecordSet, erro
 		}
 		currentRecordInfo.consumed = true
 
+		// Cleanup record name and ensure it ends with domain.ext[dot] even if dns_challenge_override_domain is set
+		// trimming both zone & domainName is probably overzealous, but better be safe then sorry
```

Comment:
For clarity, can we add an example transformation in the comment?
Could be as simple as:
```
// for example: sub.example.org -> sub.example.org.
```

or whatever is a correct example.